srv/modules/runners/validate.py

```diff
@@ -593,21 +616,25 @@ def pillars(**kwargs):
     #options = SaltOptions()
     local = salt.client.LocalClient()
     cluster = ClusterAssignment(local)
+    
+    printer = printer = get_printer(**kwargs)
+
 
     for name in cluster.names:
-        pillar(name, **kwargs)
+        pillar(name, printer=printer, **kwargs)
+    
+    printer.print_result()
 
 
-def pillar(name = None, **kwargs):
+def pillar(name = None, printer=None, **kwargs):
     """
     Check that the pillar for each cluster meets the requirements to install
     a Ceph cluster.
     """
-
-    if name == None:
-        if 'name' in kwargs:
-            name = kwargs['name']
-
```

Comment:
I didn't understand what you were explaining.  I just tried validate without the three lines and it worked.  I have seen so many python examples of args and kwargs treated separately that I did not expect a named parameter to look like a kwargs and yet not be one.